tests/test_wsgi.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import absolute_import, unicode_literals
+
+import sys
+from unittest import TestCase
+from parameterized import parameterized
+from wsgiref.util import setup_testing_defaults
+from prometheus_client import make_wsgi_app
+
+from prometheus_client import CollectorRegistry, Counter, generate_latest
+from prometheus_client.exposition import CONTENT_TYPE_LATEST
+
+
+class WSGITest(TestCase):
+    def setUp(self):
+        self.registry = CollectorRegistry()
+        self.captured_status = None
+        self.captured_headers = None
+        # Setup WSGI environment
+        self.environ = {}
+        setup_testing_defaults(self.environ)
+
+    def capture(self, status, header):
+        self.captured_status = status
+        self.captured_headers = header
+
+    def assertIn(self, item, iterable):
+        try:
+            super().assertIn(item, iterable)
+        except:  # Python < 2.7
+            self.assertTrue(
+                item in iterable,
+                msg="{item} not found in {iterable}".format(
+                    item=item, iterable=iterable
+                )
+            )
+
+    @parameterized.expand([
```

Comment:
Please keep this explicit, rather than adding a dependency.

Reply:
Done